README.md

```diff
@@ -23,20 +23,19 @@ Welcome to Braintree's iOS SDK. This library will help you accept card and alter
 - [Pay with Venmo](https://developer.paypal.com/braintree/docs/guides/venmo/overview)
 - [Apple Pay](https://developer.paypal.com/braintree/docs/guides/apple-pay/overview)
 - [ThreeDSecure](https://developer.paypal.com/braintree/docs/guides/3d-secure/overview)
-- [Visa Checkout](https://developer.paypal.com/braintree/docs/guides/secure-remote-commerce/overview)
```

Comment:
Do we still want to link to Visa Checkout? I added LPM's but do we also want to add SEPA despite the docs being hidden? We could also add a features section here and link to Data Collector, PayPal Messaging, and Shopper Insights. What do folks think?